packages/infrastructure/kube_airbyte/scripts/vault_token_rotate.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+
+echo "Using Vault address: $VAULT_ADDR"
+echo "Using Vault role: $VAULT_ROLE"
+echo "Using namespace: $NAMESPACE"
+
+# Get the JWT token from the service account
+JWT=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+echo "JWT token length: $(echo $JWT | wc -c) characters"
+
+# Test connectivity to Vault
+echo "Testing Vault connectivity..."
+HEALTH_STATUS=$(curl -s -o /dev/null -w "%{http_code}" $VAULT_ADDR/v1/sys/health)
+echo "Vault health status: $HEALTH_STATUS"
+
+if [ "$HEALTH_STATUS" != "200" ] && [ "$HEALTH_STATUS" != "429" ] && [ "$HEALTH_STATUS" != "472" ] && [ "$HEALTH_STATUS" != "473" ]; then
+  echo "Failed to connect to Vault at $VAULT_ADDR"
+  exit 1
+fi
+
+# Login to Vault using Kubernetes auth with debug info
+echo "Authenticating with Vault at $VAULT_ADDR/v1/auth/kubernetes/login..."
+echo "Using role: $VAULT_ROLE"
+
+# Create the JSON payload for the request
+AUTH_PAYLOAD=$(
+  cat <<PAYLOAD
+{
+  "jwt": "$JWT",
+  "role": "$VAULT_ROLE",
+  "ttl": "48h"
+}
+PAYLOAD
+)
+
+# Perform the login request
+LOGIN_RESPONSE=$(curl -s \
+  --request POST \
+  --header "Content-Type: application/json" \
+  --data "$AUTH_PAYLOAD" \
+  $VAULT_ADDR/v1/auth/kubernetes/login)
+
+# Pattern: look for "client_token":"some-token", extract the token part
+CLIENT_TOKEN=$(echo $LOGIN_RESPONSE | grep -o '"client_token":"[^"]*"' | sed 's/"client_token":"//;s/"$//')
```

Comment:
I think you should use the Panfactum devshell image for this container instead of curl. (a) it is already cached on every node so startup will be immediate, (b) it will have jq and you can avoid this hack which seems pretty brittle if the json format ever changes (e.g., newlines)